### ISSUE TITLE
May I request this app for inclusion to F-Droid catalog?

### ISSUE THREAD

**Issue by sgtpep** [Collinux/minimalist-launcher#2]
Hi! Thanks for a great app! I'd love to see it in F-Droid catalog eventually. Their request for inclusion policy requires authors to be notified about it. I hope you don't mind.

**Comment by GEnergetik** [comment]
Hi!
[Light Android Launcher](https://f-droid.org/en/packages/com.github.postapczuk.lalauncher/)
This is not my application.

**Comment by sgtpep** [comment]
I like yours more than this fork and wanted to submit it to F-Droid as well. Your app is just a scrollable list of all apps on a black background and nothing more, exactly what's needed, while Light Android Launcher introduces some unnecessary styles and abstractions, and it requires me to manually add apps to the main list one by one without a way to redefine their order later and has some other quirks. Is it fine if I submit your app to F-Droid, as well?

**Comment by Collinux** [comment]
I don't mind, thanks for taking the initiative!

**Comment by sgtpep** [comment]
Thank you a lot! And thanks for this really nice app! I can't wait to see it published.

**Comment by licaon-kter** [comment]
@Collinux Any chance to get an actual Release with Tags and such?

Also, for now it fails with:
```
DEBUG: buildserver > :MinimalistLauncher:javaPreCompileRelease
DEBUG: buildserver > :MinimalistLauncher:compileReleaseJavaWithJavac
DEBUG: buildserver > :MinimalistLauncher:compileReleaseNdk NO-SOURCE
DEBUG: buildserver > :MinimalistLauncher:compileReleaseSources
DEBUG: buildserver > :MinimalistLauncher:lintVitalRelease FAILED
DEBUG: buildserver > 
DEBUG: buildserver > FAILURE: Build failed with an exception.
DEBUG: buildserver > 
DEBUG: buildserver > * What went wrong:
DEBUG: buildserver > Execution failed for task ':MinimalistLauncher:lintVitalRelease'.
DEBUG: buildserver > > Could not resolve all files for configuration ':MinimalistLauncher:lintClassPath'.
DEBUG: buildserver >    > Could not find com.android.tools.lint:lint-gradle:26.1.0.
DEBUG: buildserver >      Searched in the following locations:
DEBUG: buildserver >          file:/home/vagrant/android-sdk/extras/m2repository/com/android/tools/lint/lint-gradle/26.1.0/lint-gradle-26.1.0.pom
DEBUG: buildserver >          file:/home/vagrant/android-sdk/extras/m2repository/com/android/tools/lint/lint-gradle/26.1.0/lint-gradle-26.1.0.jar
DEBUG: buildserver >          file:/home/vagrant/android-sdk/extras/google/m2repository/com/android/tools/lint/lint-gradle/26.1.0/lint-gradle-26.1.0.pom
DEBUG: buildserver >          file:/home/vagrant/android-sdk/extras/google/m2repository/com/android/tools/lint/lint-gradle/26.1.0/lint-gradle-26.1.0.jar
DEBUG: buildserver >          file:/home/vagrant/android-sdk/extras/android/m2repository/com/android/tools/lint/lint-gradle/26.1.0/lint-gradle-26.1.0.pom
DEBUG: buildserver >          file:/home/vagrant/android-sdk/extras/android/m2repository/com/android/tools/lint/lint-gradle/26.1.0/lint-gradle-26.1.0.jar
DEBUG: buildserver >      Required by:
DEBUG: buildserver >          project :MinimalistLauncher
DEBUG: buildserver > 
DEBUG: buildserver > * Try:
DEBUG: buildserver > Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
DEBUG: buildserver > 
DEBUG: buildserver > * Get more help at https://help.gradle.org
DEBUG: buildserver > 
DEBUG: buildserver > BUILD FAILED in 4s
DEBUG: buildserver > 17 actionable tasks: 17 executed
DEBUG: buildserver > ERROR: Could not build app launcher.simple.com.simplelauncher: Build failed for launcher.simple.com.simplelauncher:1.0
```
Thoughts?

**Comment by Collinux** [comment]
@licaon-kter I published the 1.0 release based on the same code. 
If your build failure is still an issue then please create a new Issue with details on your build environment, verbose output, and what you've tried to resolve it. This issue is about publishing to the F-Droid catalog.

**Comment by IzzySoft** [comment]
@Collinux you might have missed it, but that report was about trying to build your app for publishing it at F-Droid using the fdroidserver environment. It's still failing with the very same error. References:

* [CI build logs](https://gitlab.com/IzzySoft/fdroiddata/-/jobs/950331800)
* the [F-Droid packaging request](https://gitlab.com/fdroid/rfp/-/issues/1086)
* the [current merge request](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/8071)

So are you still interested, or shall we just close the task on our end?
